presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -44,6 +44,9 @@ type StageNodeInfo = {
 
 class StageStatistics extends React.Component<StageStatisticsProps, StageStatisticsState> {
     static getStages(queryInfo): Map<string, StageNodeInfo> {
+        if (!queryInfo.outputStage) {
```

Comment:
was this a bugfix? If so, please extract to separate commit

Reply:
Yes it was, but it added an error fixed in here https://github.com/prestosql/presto/pull/6214/commits/31f7a7b6095caa1eed2de3937c9250ca12609557